imageio/plugins/_freeimage.py

```diff
@@ -953,28 +953,20 @@ def set_image_data(self, array):
             n_channels = shape[0]
         
         def n(arr):  # normalise to freeimage's in-memory format
-            return arr.T[:, ::-1]
+            return arr[::-1].T
```

Comment:
In the old code the T would reverse the axis order and then flip the second axis (x axis).
The new version flips the first axis (the y axis) and then reverses the axis order.